niworkflows/viz/utils.py

```diff
@@ -459,3 +459,146 @@ def compose_view(bg_svgs, fg_svgs, ref=0, out_file='report.svg'):
         with open(out_file, 'w' if PY3 else 'wb') as f:
             f.write('\n'.join(svg))
     return out_file
+
+def transform_to_2d(data, max_axis):
```

Comment:
A one-liner docstring for this method saying what is the idea of it would be great :)